src/sentry/api/serializers/models/organization_member.py

```diff
@@ -34,3 +36,42 @@ def serialize(self, obj, attrs, user):
             'dateCreated': obj.date_added,
         }
         return d
+
+
+class OrganizationMemberWithTeamsSerializer(OrganizationMemberSerializer):
+    def get_attrs(self, item_list, user):
+        attrs = super(OrganizationMemberWithTeamsSerializer,
+                      self).get_attrs(item_list, user)
+
+        member_team_map = OrganizationMemberTeam.objects.values(
+            'organizationmember', 'team')
+
+        teams = {t.id: t for t in Team.objects.filter(
+            id__in=[item['team'] for item in member_team_map])}
+        results = defaultdict(list)
+
+        # results is a map of member id -> team_slug[]
+        for m in member_team_map:
+            results[m['organizationmember']].append(teams[m['team']].slug)
+
+        for item in item_list:
+            teams = results[item.id] if item.id in results else []
+            try:
+                attrs[item]['teams'] = teams
+            except KeyError:
+                attrs[item] = {
+                    'teams': teams
+                }
+
+        return attrs
+
+    def serialize(self, obj, attrs, user):
+        d = super(OrganizationMemberWithTeamsSerializer,
+                  self).serialize(obj, attrs, user)
+
+        try:
+            d['teams'] = attrs['teams']
```

Comment:
maybe default to an empty list if there are no teams? seems like that might make the api more consistent